style.css

```diff
@@ -1,4 +1,15 @@
 .dark-button {
+  color: red;
+}
 
-    color: red;
-}
\ No newline at end of file
+.responsive {
+  width: 100%;
+  height: auto;
+}
+
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
This is an example inline comment. :slightly_smiling_face:

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink:

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.

Reply:
tyvm, added two inline comments, in a polite way.